File: bindings/python/examples/basic_usage.py
#!/usr/bin/env python3
"""
Basic usage example for Wavemark Python bindings.
"""

import wavemark_python

def main():
    print("Wavemark Python Example")
    print("======================")
    
    # Simple function call
    message = wavemark_python.hello_world()
    print(f"Message: {message}")

if __name__ == "__main__":
    main()

File: bindings/typescript/examples/basic_usage.ts
/**
 * Basic usage example for Wavemark TypeScript bindings.
 */

import { hello_world } from '../pkg/wavemark_typescript';

function main() {
    console.log('Wavemark TypeScript Example');
    console.log('===========================');
    
    // Simple function call
    const message = hello_world();
    console.log(`Message: ${message}`);
}

// Run the example
main();